policy-suggest-dns.yaml

```diff
@@ -0,0 +1,62 @@
+APIVersion: 1
+label: recipe:policy-suggest:dns
+data:
+  recipes:
+    - name: Create DNS rule
+      description: Creates a basic DNS rule for the current namespace.
+      label: recipe:policy-suggest:dns
+      metadata:
+        - "@aporeto:author=aporeto"
+      associatedTags:
+        - "aporeto:recipe:placement=policy-suggest"
+        - "aporeto:recipe:identity=network-service-rules"
```

Comment:
`aporeto:recipe:identity` can be removed

`aporeto:recipe:placement` should is good and we could also indicate the tab: 

* "aporeto:recipe:placement=policy-suggest"
* "aporeto:recipe:placement=infrastructure"

We can check with @vjuux what he would need